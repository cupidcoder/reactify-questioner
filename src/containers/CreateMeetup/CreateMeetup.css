.CreateMeetup {
  min-height: 70vh;
}
