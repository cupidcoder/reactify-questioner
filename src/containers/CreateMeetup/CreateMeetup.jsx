import React from 'react';
import MeetupForm from '../../components/MeetupForm/MeetupForm';
import style from './CreateMeetup.css';

const CreateMeetup = () => (
  <div className={style.CreateMeetup}>
    <MeetupForm />
  </div>
);

export default CreateMeetup;
