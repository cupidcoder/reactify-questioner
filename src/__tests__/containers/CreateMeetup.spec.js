import React from 'react';
import { shallow } from 'enzyme';
import CreateMeetup from '../../containers/CreateMeetup/CreateMeetup';

describe('CreateMeetup', () => {
  it('Should correctly save snapshot of the CreateMeetup', () => {
    const wrapper = shallow(<CreateMeetup />);
    expect(wrapper).toMatchSnapshot();
  });
});
