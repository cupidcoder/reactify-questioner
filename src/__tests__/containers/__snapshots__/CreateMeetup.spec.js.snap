// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateMeetup Should correctly save snapshot of the CreateMeetup 1`] = `ShallowWrapper {}`;
